examples/copy.go

```diff
@@ -5,17 +5,32 @@
 package main
 
 import "github.com/intel-go/yanff/flow"
```

Comment:
Order of the imports should be universal: system then local,  within parenthesis